### ISSUE TITLE
toggle all on/off

### ISSUE THREAD

**Issue by jtoy** [mlcontests/mlcontests.github.io#128]
thank you for the great work, its a little confusing, i think there should be a UI to turn off/on all the GPU types and also in the headers show clearly which ones are on/off.

**Comment by hcarlens** [comment]
Thanks @jtoy! Yeah great suggestions - the UI is still very much a work in progress, I'm hoping to improve on it soon. 

I've added a "hide all" option at the top which should hopefully help . 

Making the header states clearer isn't easy right now since I'm using a third-party table library which I need to move away from. Same for showing all because there's some priority logic around which ones get shown when there's limited space. 

I'm glad this page is useful to you and I'm open to other suggestions! 

PS: I noticed that the github repo link at the bottom of the page was outdated - this page now resides in https://github.com/cloud-gpus/cloud-gpus.github.io.
